tests/testDipoleFitter.py

```diff
@@ -30,14 +30,15 @@
 from lsst.ip.diffim.dipoleFitTask import (DipoleFitAlgorithm, DipoleFitTask)
 import lsst.ip.diffim.utils as ipUtils
 
-# This file produces three unit tests of the DipoelFitAlgorithm and its related tasks and plugins.
-# Each test generates a fake image with two synthetic dipoles as input data.
-#
-# DipoleFitTest.testDipoleAlgorithm - tests the DipoleFitAlgorithm directly and ensures that the recovered
-#   fluxes and centroids are close to the input values.
-# DipoleFitTest.testDipoleTask - tests the DipoleFitTask on identical synthetic data, after running
-#   dipole detection on the image.
-# DipoleFitTest.testDipoleEdge - ensures correct handling of dipoles too close to the edge of the image.
+"""This file contains three unit tests of the DipoelFitAlgorithm and its related tasks and plugins.
+Each test generates a fake image with two synthetic dipoles as input data.
+
+DipoleFitTest.testDipoleAlgorithm - tests the DipoleFitAlgorithm directly and ensures that the recovered
+   fluxes and centroids are close to the input values.
+DipoleFitTest.testDipoleTask - tests the DipoleFitTask on identical synthetic data, after running
+   dipole detection on the image.
+DipoleFitTest.testDipoleEdge - ensures correct handling of dipoles too close to the edge of the image.
+"""
```

Comment:
I suggest removing the last 6 lines and on the doc strings for the test methods that you already helpfully provide. That way the comments are easier to keep correct as the code evolves.

The overview above that is much appreciated.